### PR TITLE
Fix #623, Replace OS_FS_* return codes with common definitions

### LINKED ISSUE #623: Replace OS_FS_* return codes with common definitions
**Is your feature request related to a problem? Please describe.**
There is mixed use of OS_* and OS_FS_* return codes.   

**Describe the solution you'd like**
Suggest deprecation of the OS_FS_ERR_* defines and switch to common definitions.

**Additional context**
nasa/osal#262
nasa/osal#118

**Requester Info**
Dan Knutsen
NASA/Goddard

### PR DESCRIPTION
**Pull request by dmknutsen**
Describe the contribution
Removed The following OS_FS* defines from osapi-os-filesys.h:
OS_FS_SUCCESS, OS_FS_ERROR , OS_FS_ERR_INVALID_POINTER, OS_FS_ERR_NO_FREE_FDS , OS_FS_ERR_INVALID_FD, and OS_FS_UNIMPLEMENTED

Fixes #623

Used sed to perform a bulk find/replace on each one to swap with common definitions.

Testing performed
Steps taken to test the contribution:

    Ran unit tests.

Expected behavior changes
None

System(s) tested on
Oracle VM VirtualBox
OS: ubuntu-19.10
Versions: cFE 6.7.12.0, OSAL 5.0.11.0, PSP 1.4.7.0

**Contributor Info **
Dan Knutsen
NASA/Goddard

### CODE REVIEW

File: fsw/cfe-core/src/inc/cfe_error.h [hunk 1]
```diff
@@ -808,12 +808,12 @@
 #define CFE_OS_ERR_INVALID_PRIORITY     (OS_ERR_INVALID_PRIORITY) /**< @copydoc OS_ERR_INVALID_PRIORITY */
 #define CFE_OS_ERROR_TASK_ID            (OS_ERROR_TASK_ID) /**< @brief This doesn't actually exist */
 #define CFE_OS_SEM_UNAVAILABLE          (OS_SEM_UNAVAILABLE) /**< @brief This doesn't actually exist */
-#define CFE_OS_FS_ERROR                 (OS_FS_ERROR) /**< @copydoc OS_FS_ERROR */
-#define CFE_OS_FS_ERR_INVALID_POINTER   (OS_FS_ERR_INVALID_POINTER) /**< @copydoc OS_FS_ERR_INVALID_POINTER */
+#define CFE_OS_ERROR                 (OS_ERROR) /**< @copydoc OS_ERROR */
```
Comment (skliper) [inline fsw/cfe-core/src/inc/cfe_error.h:811]: Leave as CFE_OS_FS_ERROR, this whole group is deprecated in #582 but need to deprecate before removing.  Same for CFE_OS_FS_ERR_INVALID_POINTER.

File: fsw/cfe-core/src/tbl/cfe_tbl_internal.h [hunk 2]
```diff
@@ -380,12 +380,12 @@ int32   CFE_TBL_GetWorkingBuffer(CFE_TBL_LoadBuff_t **WorkingBufferPtr,
 **                              of table image file to be loaded
 **
 ** \retval #CFE_SUCCESS                      \copydoc CFE_SUCCESS
-** \retval #OS_FS_ERR_INVALID_POINTER        \copydoc OS_FS_ERR_INVALID_POINTER      
+** \retval #OS_INVALID_POINTER        \copydoc OS_INVALID_POINTER
```
Comment (skliper) [inline fsw/cfe-core/src/tbl/cfe_tbl_internal.h:383]: Make spacing consistent

**Comment by astrogeco** [comment]
CCB 20200422 - APPROVED

**Comment by astrogeco** [comment]
@dmknutsen check conflicts, might want to rebase

**Pull request merged**